SensorNode/sensor_nodes/remoteSensorNode.py

```diff
@@ -36,6 +38,11 @@ def __init__(self, config_path: str):
         self.ip_camera_port = ip_camera_params['port']
         assert(isinstance(self.ip_camera_port, int))
 
+        if os.getuid() == 0:
+            self.ff_log_dir = os.path.abspath(os.path.join('var', 'log', 'ffmpeg_logs'))
+        else:
+            self.ff_log_dir = os.path.join(appdirs.user_log_dir('ASMSensorNode'), 'ffmpeg_logs')
```

Comment:
We should be using `pathlib.Path` for this, see corresponding comment in ASM-data-server